### PR TITLE
Implement module RegisterLegacyAminoCodec

### PR DESCRIPTION
**Pull request by anhductn2001**

### CODE REVIEW

File: x/feeabs/module.go [hunk 1]
```diff
@@ -43,10 +43,13 @@ func (AppModuleBasic) Name() string {
 	return types.ModuleName
 }
 
+func (AppModuleBasic) RegisterCodec(cdc *codec.LegacyAmino) {
```
Comment (GNaD13) [inline x/feeabs/module.go]: Do we need this func here? I think sdk never call this func

Comment (anhductn2001) [reply x/feeabs/module.go]: Because I see it in TODO list.

Comment (GNaD13) [reply x/feeabs/module.go]: Hmmm. weird. I think we just need to implement this func `RegisterLegacyAminoCodec`

Comment (GNaD13) [reply x/feeabs/module.go]: I think we can just remove this func. It not work in sdk

**Comment by anhductn2001** [comment]
Can you review it again @GNaD13 ??

**Review by GNaD13 — APPROVED** [review]

**Pull request merged**